apps/term-apply/pkg/server/server.go

```diff
@@ -25,7 +25,7 @@ type Server struct {
 }
 
 func NewServer(c Config) (*Server, error) {
-	am, err := applicant.NewApplicantManager(c.csvTmpFile, c.resumeTmpDir, c.s3Bucket, c.s3ResumePrefix, c.s3CsvPrefix)
+	am, err := applicant.NewApplicantManager(c.resumeTmpDir, c.s3Bucket, c.s3ResumePrefix, c.dynamodbTable, c.dynamodbIndex)
```

Comment:
I left this behind, so not relevant to this PR, but eventually we should just feed the entire config to NewApplicantManager as a single argument vs 5 distinct values.

Reply:
I was wondering about this. The reason I didn't change it in this PR was that all the values for the config struct are private and I wasn't sure if there was a hard reason for that. It would make things a lot cleaner looking for some things, but yeah not super relevant to this one to fix now.

Reply:
they aren't private for any practical reason that I can think of or remember. lol